rs/src/gstd/calls.rs

```diff
@@ -14,6 +15,7 @@ pub struct GStdArgs {
     reply_deposit: Option<GasUnit>,
     #[cfg(not(feature = "ethexe"))]
     reply_hook: Option<Box<dyn FnOnce() + Send + 'static>>,
+    redirect_on_exit: bool,
```

Comment:
to be conscious about this param and it's consequences: using it can cause for an original message **M** to have more gas burned than without it. That's because redirect happens after receiving the response from the sent message, which took some gas from the **M**.

just noticing, this info could be beneficial to add to docs

Reply:
totally worth mentioning in the documentation - and how it works